package.json

```diff
@@ -101,5 +102,12 @@
     "packages/validator-ajv8",
     "packages/snapshot-tests",
     "packages/shadcn"
-  ]
+  ],
+  "dependencies": {
+    "@restart/hooks": "^0.6.2",
+    "@restart/ui": "^1.9.4",
+    "ajv": "^8.17.1",
+    "ajv-keywords": "^5.1.0",
+    "uncontrollable": "^9.0.0"
+  }
```

Comment:
These belong as `devDependencies` in your theme's `package.json`. If you need the `@restart` stuff as dependencies in your theme, do that